scripts/docgen/toc.yaml

```diff
@@ -0,0 +1,145 @@
+toc:
+- title: "firebase"
+  path: /docs/reference/js/firebase
```

Comment:
How is the path determined?

Reply:
The directory is the one in google3, every file is at the top level, the file naming scheme is generated by Typedoc based on the hierarchy pulled from index.d.ts.

Reply:
Ah I see. I'm wondering if we need to put any google3 related process in google3.

@samtstern Is it an acceptable practice to expose google3 directory structure to the public?

Reply:
Yeah as long as it's not full of codenames or PII (like LDAPs) you don't have to worry about it.